frontend/src/__tests__/cypress/cypress/tests/mocked/hardwareProfiles/legacyHardwareProfiles.cy.ts

```diff
@@ -0,0 +1,612 @@
+      disabledRow.findEnabled().should('be.disabled');
+    });
+  });
+
+  describe('Accelerator profiles', () => {
+    const testProfileName = 'test-accelerator-profile';
+    const testProfileDisplayName = 'Test Accelerator Profile';
+
+    beforeEach(() => {
+      cy.interceptK8sList(
+        { model: AcceleratorProfileModel, ns: testNamespace },
+        mockK8sResourceList([
+          mockAcceleratorProfile({
+            name: testProfileName,
+            displayName: testProfileDisplayName,
+            namespace: testNamespace,
+          }),
+        ]),
+      );
+    });
+
+    it('should delete', () => {
+      cy.interceptK8s(
+        'DELETE',
+        {
+          model: AcceleratorProfileModel,
+          ns: testNamespace,
+          name: testProfileName,
+        },
+        mock200Status({}),
+      ).as('delete');
+
+      legacyHardwareProfile.visit();
+      legacyHardwareProfile.findExpandButton().click();
+      legacyHardwareProfile.getRow(testProfileDisplayName).findKebabAction('Delete').click();
+
+      deleteModal.findSubmitButton().should('be.disabled');
+      deleteModal.findInput().fill('Delete 3 resources');
+      deleteModal.findSubmitButton().should('be.enabled').click();
+
+      cy.wait('@delete');
+    });
+
+    it('should duplicate', () => {
+      legacyHardwareProfile.visit();
+      legacyHardwareProfile.findExpandButton().click();
+      legacyHardwareProfile.getRow(testProfileDisplayName).findKebabAction('Duplicate').click();
+
+      cy.url().should('include', `/hardwareProfiles/duplicate/${testProfileName}`);
+    });
+
+    it('should edit', () => {
+      legacyHardwareProfile.visit();
+      legacyHardwareProfile.findExpandButton().click();
+      legacyHardwareProfile.getRow(testProfileDisplayName).findKebabAction('Edit').click();
+
+      cy.url().should('include', `/hardwareProfiles/edit/${testProfileName}`);
+
+      editHardwareProfile.findMigrationAlert().should('be.visible');
+    });
+
+    it('should migrate', () => {
+      cy.interceptK8s(
+        'DELETE',
+        {
+          model: AcceleratorProfileModel,
+          ns: testNamespace,
+          name: testProfileName,
+        },
+        mock200Status({}),
+      ).as('deleteSource');
+
+      cy.interceptK8s(
+        'POST',
+        { model: HardwareProfileModel },
+        mockHardwareProfile({
+          name: testProfileName,
+          displayName: testProfileDisplayName,
+        }),
+      ).as('createTarget');
+
+      legacyHardwareProfile.visit();
+      legacyHardwareProfile.findExpandButton().click();
+      legacyHardwareProfile.getRow(testProfileDisplayName).findKebabAction('Migrate').click();
+
+      migrationModal.findSubmitButton().should('be.disabled');
+      migrationModal.findInput().fill(testProfileDisplayName);
+      migrationModal.findSubmitButton().should('be.enabled').click();
+
+      cy.wait(['@deleteSource', '@createTarget', '@deleteSource', '@createTarget']);
```

Comment:
@caponetto just a small addition here, Can you assert the payload of `@createTarget` as mentioned in this [doc](https://github.com/opendatahub-io/odh-dashboard/blob/main/docs/testing.md#:~:text=When%20a%20payload%20is%20sent%20as%20part%20of%20the%20network%20request%2C%20the%20test%20should%20assert%20the%20payload%20is%20the%20expected%20value.%20For%20example%20after%20filling%20out%20and%20submitting%20a%20form%2C%20assert%20the%20form%20values%20are%20present%20in%20the%20network%20request%3A)

Reply:
Done in https://github.com/opendatahub-io/odh-dashboard/pull/3959/commits/78f7c8c58cc68243914daa83aa927c77ef859d7d